src/audio/2e230_len_2190.c

```diff
@@ -21,7 +21,9 @@ void func_80052E30(u8 index) {
     voice->priority = AU_PRIORITY_FREE;
 }
 
-#ifndef SHIFT
+#ifdef SHIFT
```

Comment:
I think we can probably get rid of these shifting asm. It was good for proving the ROM segments were shiftable, but now that we've finished that, its probably fine to get rid of these sometime soon and just use the ifdef on the SBN offset.